### ISSUE TITLE
hashpound (header) not highlighted

### ISSUE THREAD

**Issue by rodhash** [plasticboy/vim-markdown#458]
Hi

This is my first issue being opened so if I miss some detail pls bear with me :)

This vim-markdown is simply awesome! However something is bugging me for awhile, cant find out why this happens.

See below:  the hash pound does not get highlighted, I've already tried removing everything from my .vimrc working with a minimal config but the issue persists.

![image](https://user-images.githubusercontent.com/29671981/67836431-60273c80-facb-11e9-96b5-e6f8e41b6043.png)

Any idea / suggestion pls?

Both my vim, tmux and everything are up-to-date, compiled the lastest versions. Also tried in my work notebook running an older version of vim (7.4) and the same thing happens.

Not really sure why exactly the **#** is left without the same color as the header.

As we can see the vim recognizes the correct filetype (markdown), also other kind of headers such as using **====** or **-----** works just fine.

The issue is only with the hash pound itself.

Thanks!

**Comment by cup** [comment]
https://github.com/plasticboy/vim-markdown/issues/388

https://github.com/plasticboy/vim-markdown/issues/458
https://github.com/plasticboy/vim-markdown/issues/461
https://github.com/plasticboy/vim-markdown/issues/471
https://github.com/plasticboy/vim-markdown/issues/74

**Comment by rodhash** [comment]
Hi

After a long time I found myself searching again about this and I figured this is a silly option that allow us to customize the colors by updating the .vimrc.

In my case I'm not quite sure why the hashpound is not highlighting itself .. I've tested this in many different OS (mainly Redhat, Fedora and Ubuntu) and DE (mostly KDE but also Gnome, xfce and i3).

This is so silly but I was able to customize with the following two lines:

`    autocmd FileType markdown highlight mkdHeading cterm=bold ctermfg=5
    autocmd FileType markdown highlight htmlH1 cterm=bold ctermfg=5`


![image](https://user-images.githubusercontent.com/29671981/77599569-e82b3a00-6ee3-11ea-8ff1-80c28a85adeb.png)


Not sure if I contributed with anything but I just remembered to come back here to register what was missing here.

Cheers
